test/runtests.jl

```diff
@@ -46,4 +46,7 @@ using SafeTestsets
     if !Sys.isapple()
         @time @safetestset "Graphs" begin include("visualization/graphs.jl") end
     end
+    
+    ### Tests extensions. ###
+    @time @safetestset "Homotopy Continuation Extension" begin include("extensions/homotopy_continuation.jl") end
```

Comment:
The test probably also needs a check that the extension mechanism exists to run conditionally then?

Reply:
I think this should work, given the work around (but should hold my horses until the 1.6 test actually passes).

However, an alternative would be to, for extensions, simply disable those tests on 1.6. I will let you decide.

Currently, I added 
```julia
if !isdefined(Base, :get_extension) # Workaround for 1.6 compatibility.
    include("../ext/CatalystHomotopyContinuationExtension.jl")
end
```
to Catalyst.jl to make things work on 1.6.

Reply:
Indeed I spoke to quickly. Is disabling the test for this case as you mentioned a working solution?

Reply:
Yeah, and going forward we can just do the same for any future extensions.